js/models/profile/Profile.js

```diff
@@ -113,6 +113,11 @@ export default class extends BaseModel {
       addError('website', app.polyglot.t('profileModelErrors.provideValidURL'));
     }
 
+    if (typeof attrs.vendor !== 'boolean') {
+      // this error should never be visible to the user
+      addError('vendor', `The vendor is invalid: ${attrs.vendor}`);
```

Comment:
It would be nice if the error message informed you why it's invalid... something like 'Please provide a vendor as a boolean".